client/signup/steps/theme-selection/index.jsx

```diff
@@ -59,7 +48,8 @@ module.exports = React.createClass( {
 	},
 
 	getThemes() {
-		return this.props.signupDependencies.themes || this.props.themes;
+		const dependencies = this.props.signupDependencies;
+		return dependencies.themes || getThemes( dependencies.surveyQuestion, dependencies.designType );
```

Comment:
Oh, I guess it would just fail the filters and you'd end up with the default themes. Ok.